src/main.py

```diff
@@ -232,8 +235,35 @@ async def info(interaction: Interaction) -> None:
         return
 
     await bc.send_stats(interaction)
-
-
+    
+# Preview command
+@client.slash_command(dm_permission=False)
+async def preview(
+    interaction: Interaction,
+    uploaded_file: Attachment = SlashOption(description="The uploaded file for submission."),
+    submit_message: str = SlashOption(required=False, description="The text for submission message.")
+) -> None:
+    """Send a message to user with embedded preview of uploaded song's metadata."""
+    user = interaction.user
+    attachment_filepath = filepath_builder(interaction.id, uploaded_file)
+    await uploaded_file.save(fp=attachment_filepath)
+    
+    # TODO: Catch file types that are not media files containing .wav, .mp3, etc.
+    # For now, we'll just have if file is None.
+    if uploaded_file.content_type is None:
```

Comment:
There's no TODO here, we should make preview parsing follow the same logic as submission parsing on launch. in discord_utils.py we can see that the exact condition for whether we try to parse a file is 

```
            if attachment.content_type is None or (
                not attachment.content_type.startswith("audio")
                and not attachment.content_type.startswith("video")
            ):
```

any other invalidities should be caught at a later stage. This if-statement is complex enough that i'd probably extract it out into a function